commit 2d79ffee423fd4570599258e00689cc745e8785e
Author: Nicolas Morey <nmorey@suse.com>
Date:   Fri Nov 7 17:19:54 2025 +0100

    UCP/CORE: Fix config type for dynamic_tl_progress_factor
    
    Signed-off-by: Nicolas Morey <nmorey@suse.com>

diff --git src/ucp/core/ucp_context.c src/ucp/core/ucp_context.c
index 8b9dbeaca9ea..4cbae096ed93 100644
--- src/ucp/core/ucp_context.c
+++ src/ucp/core/ucp_context.c
@@ -440,7 +440,7 @@ static ucs_config_field_t ucp_context_config_table[] = {
    "Number of usage tracker rounds performed for each progress operation. Must be\n"
    "non-zero value.",
    ucs_offsetof(ucp_context_config_t, dynamic_tl_progress_factor),
-   UCS_CONFIG_TYPE_TIME_UNITS},
+   UCS_CONFIG_TYPE_UINT},
 
   {"RESOLVE_REMOTE_EP_ID", "n",
    "Defines whether resolving remote endpoint ID is required or not when\n"
